x-pack/plugins/alerts/server/alert_type_registry.ts

```diff
@@ -157,6 +179,9 @@ export class AlertTypeRegistry {
           actionVariables,
           producer,
           minimumLicenseRequired,
+          enabledInLicense:
+            this.licenseState.isLicenseValidForAlertType(id, name, minimumLicenseRequired)
+              .isValid === true,
```

Comment:
nit.
Since this is a boolean, the `===` seems redundant. :)
```suggestion
            this.licenseState.isLicenseValidForAlertType(id, name, minimumLicenseRequired)
              .isValid,
```